src/ManageCourses.Api/Model/UcasCourseEnrichmentGetModel.cs

```diff
@@ -16,5 +16,7 @@ public UcasCourseEnrichmentGetModel()
         public int CreatedByUserId { get; set; }//TODO should be a user object however it causes an error 
         public int UpdatedByUserId { get; set; }//TODO should be a user object however it causes an error 
         public EnumStatus Status { get; set; }
+        public string InstCode { get; set; }
+        public string CourseCode { get; set; }
     }
 }
```

Comment:
This is virtually identical to the UcasInstitutionEnrichmentGetModel. Can we create a base class for these types

Reply:
card it up ;-)